### ISSUE TITLE
Odd extra tabstop on setting

### ISSUE THREAD

**Issue by mjbvz** [microsoft/vscode#167629]
Testing #167084

1. Follow the steps from #167084
2. In `emmet.extensionsPath`, keep tabbing a few more times

**Bug**
One of the tab stops selects this horizontal line element:

<img width="1462" alt="Screenshot 2022-11-29 at 12 53 27 PM" src="https://user-images.githubusercontent.com/12821956/204645552-662b61ae-e84a-4053-9fb1-36b03da53126.png">

I think this is an empty list but it is odd that it gets selected here

**Comment by eleanorjboyd** [comment]
I am still seeing this issue occurring in the most recent version of insiders

<img width="1051" alt="Screenshot 2022-12-01 at 2 06 42 PM" src="https://user-images.githubusercontent.com/26030610/205169218-67307446-047c-40f7-969c-f301a55b5538.png">

**Comment by rzhao271** [comment]
Unable to repro. Moving to January for verification.
